Adds fixed table layout to table block

karmatosed opened the pull request:
I have added this to both the editor and also to the front styles. It might be a point for discussion along with should this be done, but also would the editor and theme show it?

Fixes #16045

Commit:
Adds fixed table layout to table block

I have added this to both the editor and also to the front styles. It might be a point for discussion along with should this be done, but also would the editor and theme show it?

Fixes #16045

talldan commented:
Hey @karmatosed - what are your thoughts on the existing option for Fixed Table Cells:
![Screen Shot 2019-12-03 at 11 36 28 am](https://user-images.githubusercontent.com/677833/70018497-49d83900-15c1-11ea-93e0-fcc207a5629d.png)

Is that something you think could be removed?

karmatosed commented:
@talldan I do and let me remove that from the PR.

ZebulanStanphill commented on packages/block-library/src/table/editor.scss:
```diff
@@ -6,6 +6,7 @@
 		height: auto;
 
 		table {
+			table-layout: fixed;
```
Isn't this unnecessary since `style.scss` is loaded in the editor?

karmatosed replied:
In testing it was for me, feel free to iterate on PR if you don't find it is in testing @ZebulanStanphill

Commit:
Removing the option

This builds on previous PR and removes option for fixed width, as it's now default.

talldan reviewed (commented):
@karmatosed This looks pretty close, a few things around the tests and deprecations might need a bit of extra work. Let me know if you'd like me to take a look at those few items.

Commit:
Changes from review

karmatosed commented:
@talldan what do you think about closing this as it's sat so long? I can always close this and we could start a fresh PR, if easier.

annezazu commented:
Closing this out -- welcome folks to open a new one to keep it fresh, as said above!
